src/jqLite.js

```diff
@@ -671,7 +671,10 @@ forEach({
       }
       return element.value;
     }
-    element.value = value;
+    // Workaround for Firefox validation #12102.
+    if (element.value != value) {
```

Comment:
Hm...although that is not accurate for `value === null`, because `String(null) === 'null'`, but `element.value = null ==> element.value === ''`, so that need to be taken into account as well.